.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+      
+    steps:
+    
+      - name: Checkout Repository
+        uses: actions/checkout@master
+
+      - name: Setup Crystal
+        uses: MeilCli/setup-crystal-action@master
+        with: 
+          crystal_version: 1.0.0
+          shards_version: latest
+
+      - name: Install dependencies
+        run: shards install --ignore-crystal-version
+
+      - name: Check Formatting
+        run: crystal tool format --check
+
+      - name: Run Ameba
+        run: ./bin/ameba
+
+      - name: Start Redis
+        uses: superchargejs/redis-github-action@master
+        with:
+          redis-version: 6
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+
+      - name: Install Redis Gem
+        run: gem install redis
```

Comment:
Can we remove the need for Ruby and this gem? I don’t recall why it is needed.

Reply:
I think they are needed because there are some tests which check the Ruby compatibility.